applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -123,25 +123,23 @@ static void pa_synced_cb(struct bt_audio_broadcast_sink *sink, struct bt_le_per_
 static void pa_sync_lost_cb(struct bt_audio_broadcast_sink *sink)
 {
 	int ret;
-	struct event_t event;
 
 	if (broadcast_sink == NULL) {
 		LOG_ERR("Unexpected PA sync lost");
 		return;
 	}
 
-	LOG_WRN("Sink disconnected");
+	LOG_INF("Sink disconnected");
 
+	synced_to_broadcast = false;
```

Comment:
Why set this to false here, then call reset that will set this to false if already true? Will sink_stop return error if sync_lost?

Reply:
In this situation the broadcast source is already stopped, and bt_audio_broadcast_sink_stop() returns an error code if it is called. I can write a comment explaining this